sklearnex/svm/_common.py

```diff
@@ -100,27 +224,38 @@ def _compute_balanced_class_weight(self, y):
         return recip_freq[le.transform(classes)]
 
     def _fit_proba(self, X, y, sample_weight=None, queue=None):
+        from .._config import config_context, get_config
```

Comment:
Runtime import is not necessary overhead, especially, when `.._config` is not cached.

Reply:
I guess that was a mistake from the original design (taken directly from #1361), will change it.